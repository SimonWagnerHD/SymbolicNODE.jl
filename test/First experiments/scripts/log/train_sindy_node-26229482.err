  Activating project at `~/home/SymbolicNODE.jl`
