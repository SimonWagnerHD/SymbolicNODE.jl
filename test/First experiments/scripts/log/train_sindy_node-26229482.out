Begin training
Epoch: 5; Train Loss: 0.9699205; Validation Loss: 0.9722092
Epoch: 10; Train Loss: 0.85154134; Validation Loss: 1.004021
Epoch: 15; Train Loss: 0.76583135; Validation Loss: 1.027201
Epoch: 20; Train Loss: 0.6977906; Validation Loss: 1.0351242
Epoch: 25; Train Loss: 0.65293443; Validation Loss: 1.0464865
Epoch: 30; Train Loss: 0.61749375; Validation Loss: 1.0580887
Epoch: 35; Train Loss: 0.56870043; Validation Loss: 1.0529613
Epoch: 40; Train Loss: 0.550763; Validation Loss: 1.07047
Epoch: 45; Train Loss: 0.53187454; Validation Loss: 1.1163783
Epoch: 50; Train Loss: 0.52001935; Validation Loss: 1.1442808
Epoch: 55; Train Loss: 0.49611402; Validation Loss: 1.1761175
Epoch: 60; Train Loss: 0.48494846; Validation Loss: 1.1907814
Epoch: 65; Train Loss: 0.45776188; Validation Loss: 1.1844572
Epoch: 70; Train Loss: 0.45149913; Validation Loss: 1.1926829
Epoch: 75; Train Loss: 0.44019982; Validation Loss: 1.1779021
Epoch: 80; Train Loss: 0.43434754; Validation Loss: 1.1757509
Epoch: 85; Train Loss: 0.43008876; Validation Loss: 1.1929059
Epoch: 90; Train Loss: 0.42176545; Validation Loss: 1.1861074
Epoch: 95; Train Loss: 0.40885; Validation Loss: 1.1597391
Epoch: 100; Train Loss: 0.40681285; Validation Loss: 1.1631871
